.evergreen/config.yml

```diff
@@ -1431,14 +1439,6 @@ tasks:
       - func: run-aws-auth-test-with-aws-EC2-credentials
       - func: run-aws-ECS-auth-test
 
-  - name: go1.10-build-cse
```

Comment:
This task is no longer needed because the `build` target in the `Makefile` now respects the tags from the `BUILD_TAGS` environment variable so it compiles the files for the `cse` tag.